packages/flutter_tools/lib/src/commands/devices.dart

```diff
@@ -12,6 +12,14 @@ import '../globals.dart' as globals;
 import '../runner/flutter_command.dart';
 
 class DevicesCommand extends FlutterCommand {
```

Comment:
The Devices command also needs a test for this new functionality, that passing the flag outputs parsable JSON. You'll need to mock the device manager with a fake device so you know what to expect